src/index.js

```diff
@@ -53,99 +94,12 @@ export { default as TableHeader } from './DataTable/TableHeaderRow';
 
-// Pass through components from react-bootstrap
-
-/*
-  TODO: Accordion
-  - a11y defect: Trigger button doesn't show expanded/collapsed state. Issue logged some time ago.
-
-export { default as Accordion } from 'react-bootstrap/Accordion';
-export { default as AccordionContext } from 'react-bootstrap/AccordionContext';
-export { default as AccordionCollapse } from 'react-bootstrap/AccordionCollapse';
-export {
-  default as AccordionToggle,
-  useAccordionToggle,
-} from 'react-bootstrap/AccordionToggle';
-
-*/
-export { default as Alert } from 'react-bootstrap/Alert';
-export { default as Badge } from 'react-bootstrap/Badge';
-/*
-  TODO: Breadcrumb
-  a11y defect: aria-current (needed semantics) is on the <li>, not on the <a>, which takes focus
-
-export { default as Breadcrumb } from 'react-bootstrap/Breadcrumb';
-export { default as BreadcrumbItem } from 'react-bootstrap/BreadcrumbItem';
-*/
-export { default as ButtonGroup } from 'react-bootstrap/ButtonGroup';
-export { default as ButtonToolbar } from 'react-bootstrap/ButtonToolbar';
-export { default as Card } from 'react-bootstrap/Card';
-export { default as CardColumns } from 'react-bootstrap/CardColumns';
-export { default as CardDeck } from 'react-bootstrap/CardDeck';
-export { default as CardImg } from 'react-bootstrap/CardImg';
-export { default as CardGroup } from 'react-bootstrap/CardGroup';
-export { default as Carousel } from 'react-bootstrap/Carousel';
-export { default as CarouselItem } from 'react-bootstrap/CarouselItem';
-export { default as CloseButton } from 'react-bootstrap/CloseButton';
-export { default as Col } from 'react-bootstrap/Col';
-export { default as Collapse } from 'react-bootstrap/Collapse';
-export { default as DropdownButton } from 'react-bootstrap/DropdownButton';
-export { default as Fade } from 'react-bootstrap/Fade';
-export { default as Form } from 'react-bootstrap/Form';
-export { default as FormControl } from 'react-bootstrap/FormControl';
-export { default as FormCheck } from 'react-bootstrap/FormCheck';
-export { default as FormFile } from 'react-bootstrap/FormFile';
-export { default as FormGroup } from 'react-bootstrap/FormGroup';
-export { default as FormLabel } from 'react-bootstrap/FormLabel';
-export { default as FormText } from 'react-bootstrap/FormText';
-export { default as Container } from 'react-bootstrap/Container';
-export { default as Image } from 'react-bootstrap/Image';
-export { default as Figure } from 'react-bootstrap/Figure';
-export { default as InputGroup } from 'react-bootstrap/InputGroup';
-export { default as Media } from 'react-bootstrap/Media';
+// Pass through any needed whole third-party library functionality
+// useTable for example is needed to use the DataTable component seamlessly
+// rather than setting a peer dependency in this project, we opt to tightly
+// couple these dependencies by passing through needed functionality.
 export { useTable } from 'react-table';
```

Comment:
We're gonna need to export more than useTable from react-table... I'll make a PR for it.